subprojects/docs/src/docs/userguide/migration/upgrading_version_7.adoc

```diff
@@ -180,6 +180,16 @@ property has been deprecated.
 Use the link:{groovyDslPath}/org.gradle.api.tasks.scala.ScalaCompile.html#org.gradle.api.tasks.scala.ScalaCompile:destinationDirectory[ScalaCompile.destinationDirectory]
 property instead.
 
+[[deprecated_flat_project_structure]]
+=== Deprecated flat project structure
+
+There are several disadvantages of using a flat project structure. One example being that Gradle linkhttps://github.com/gradle/gradle/issues/13891[file-system watching cannot be efficiently used].
```

Comment:
There's a missing : between link and https

Reply:
Done.